tests/resources/Harbor-Util.robot

```diff
@@ -20,13 +20,14 @@ Library  OperatingSystem
 *** Variables ***
 ${HARBOR_SHORT_VERSION}  0.5.0
 ${HARBOR_VERSION}  harbor_0.5.0-9e4c90e
-${MY_PROJECTS_TABLE}  div.table-body-container > table.table.table-pane
+${MY_PROJECTS_TABLE}  harbor-app > harbor-shell > clr-main-container > div > div > project > div > div > list-project
```

Comment:
We may want to ask the Admiral team (or whoever does the web UI) to include/create an id for this table.  That will help shorten this css identifier quite a bit and would be beneficial when testing the UI.
Also, `div>list-project` as an identifier works instead of the longer identifier.